src/openforms/submissions/utils.py

```diff
@@ -85,13 +85,12 @@ def send_confirmation_email(submission: Submission):
 
     content = email_template.render(submission)
 
-    try:
+    if hasattr(submission, "appointment_info"):
         client = get_client()
         content += client.get_appointment_details_html(
             submission.appointment_info.appointment_id
         )
-    except (AppointmentInfo.DoesNotExist, ValueError):
-        pass
+        content += client.get_appointment_links_html(submission)
```

Comment:
Let's stop hardcoding these things and just piling on top of content, and instead implement a custom template tag such as `{% appointment_information %}` and `{% get_appointment_links as links %}` so that these can be templated out by the end user.

The end result should be that end end-user can do something like this:

```django
{# other content such as summary #}

{% appointment_information %}

{% get_appointment_links as links %}

U kan deze afspraak <a href="{{ links.cancel_url }}">annuleren</a>
```

& then of course document these options with the rest of the templates documentation.